modules/mediaConsortiumBidAdapter.js

```diff
@@ -102,3 +158,35 @@ function isValidResponse(response) {
       isPlainObject(response.body) &&
       isArray(response.body.bids)
 }
+
+function sendEventToGA4(eventName, additionalPayload, customTimestamp) {
+  if (!additionalPayload) {
+    additionalPayload = {};
+  }
+
```

Comment:
Please add IP and GeoLocation setting.
ref) https://github.com/mediaconsortium/Prebid.js/commit/315f898cfc2c0820d6825682c0f2d3507b348f5f#diff-3274f1a37032fb0ae4e2823def0007c634e869ae0dfc304ff6a12c36513c3a52R1074-R1092